CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ CHANGELOG
 unreleased
 ----------
 - Fix issue where payment method text may be clipped
+- Add ability to let the Braintree API decide which payment method to show first (typically, the last payment method added to the customer) instead of showing the default payment method first
```

Comment:
I think this wording is a bit confusing. What determines a `default payment method`?

Reply:
Good callout. So the customer's default payment method is the _first_ payment method added to the customer, or whatever payment method is made the default using the `Custromer: update` methods or set as default in the Braintree control panel.

How about this:

```suggestion
- Add ability to let the Braintree API decide which payment method to show first (typically, the last payment method added to the customer) instead of showing the payment method designated as the customer's default in the Braintree control panel
```

Reply:
This update is helpful! Thanks for clarifying